src/circleci-base/scripts/merge-develop.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Description: Automatically merge develop branch to master on each commit
+
+GIT_COMMIT_MESSAGE=$(git --git-dir=/tmp/workspace/.git log --format=%B -n 1 "$CIRCLE_SHA1")
+if [[ $GIT_COMMIT_MESSAGE == *"[DEV]"* ]]; then
+  echo "Not merging. There a DEV prefix"
+  exit 0
+fi
```

Comment:
Added this section also here, just in case we need to stop a change from merging into master. I was thinking cases that we need to test a `post-deploy` script only on develop environments.